install/kubernetes/istio-rbac-beta.yaml

```diff
@@ -8,6 +8,12 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: istio-pilot
 rules:
+- apiGroups: ["config.istio.io"]
+  resources: ["istiokinds"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["apiextensions.k8s.io"]
```

Comment:
Do we have different roles for creating new Istio config types (via CRD) vs. creating new instances of these types? The former would be limited to installation time and the latter restricted to specific components by type.